json_schema_fuzz.py

```diff
@@ -14,7 +15,7 @@ def random_object(schema):
     required = schema.get("required", [])
     object = dict()
     for key, value in properties.items():
-        if key in required or random.randint(0, 1):
+        if key in required or random.choice([True, False]):
```

Comment:
Why?

Reply:
Lower down in the code I had used random.choice([True, False]) format and I would like to remain consistent.

Reply:
Fair enough. I think this makes it more readable too.